administration/src/bp-modules/applications/__tests__/VerificationsQuickIndicator.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { act } from 'react'
+
+import { renderWithTranslation } from '../../../testing/render'
+import VerificationQuickIndicator from '../VerificationsQuickIndicator'
+import { VerificationStatus } from '../VerificationsView'
+import {
+  verificationsAwaiting,
+  verificationsMixed,
+  verificationsRejected,
+  verificationsVerified,
+} from '../__mocks__/verificationData'
+
+describe('VerificiationQuickIndicator', () => {
+  it('should show only awaiting verifications', async () => {
+    const { getByTestId } = renderWithTranslation(<VerificationQuickIndicator verifications={verificationsAwaiting} />)
+    await act(async () => null) // Blueprint5.Icon update
+    expect(getByTestId(`indicator-${VerificationStatus.Verified}`).textContent).toBe(': 0')
+    expect(getByTestId(`indicator-${VerificationStatus.Awaiting}`).textContent).toBe(': 2')
+    expect(getByTestId(`indicator-${VerificationStatus.Rejected}`).textContent).toBe(': 0')
+  })
+
+  it('should show only mixed verifications', async () => {
```

Comment:
```suggestion
  it('should show mixed verifications', async () => {
```
:upside_down_face: or: should show verifications in all statuses